package.json

```diff
@@ -41,7 +41,7 @@
     "geolib": "^2.0.21",
     "memory-cache": "^0.1.6",
     "moment": "^2.14.1",
-    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git#feature/change-opening-times-format",
+    "moment-opening-times": "git://github.com/nhsuk/moment-opening-times.git",
```

Comment:
I've tagged the release in github so you can specify a version (e.g. git://github.com/nhsuk/moment-opening-times.git#1.0.0)